### PR TITLE
feat: SKIP_VERSION

### PR DESCRIPTION
**Pull request by rarkins**

### CODE REVIEW

**Comment by rarkins** [comment]
Any suggestions on:
- How to test this manually before merging? and
- Should we test this automatically somehow?

File: src/usr/local/buildpack/tools/git.sh [hunk 1]
```diff
@@ -14,4 +14,4 @@ curl --retry 3 -sSL \
 
 apt_install git
 
-git --version
+SKIP_VERSION || git --version
```
Comment (viceice) [inline src/usr/local/buildpack/tools/git.sh:17]:
```suggestion
$SKIP_VERSION || git --version
```
isn't it a environment variable?

**Review by viceice — APPROVED** [review]
not easily testable 🤔

**Comment by viceice** [comment]
you can test locally by building the base image, run it,  exec install tool and check the output to not contain the version twice. one will be printed by the installer

**Comment by rarkins** [comment]
Tested, found a better way:

```
root@0b204777c3ec:/# install-tool python 3.11.3
installing v2 tool python v3.11.3
linking tool python v3.11.3
Python 3.11.3
pip 23.0.1 from /opt/buildpack/tools/python/3.11.3/lib/python3.11/site-packages/pip (python 3.11)
Installed v2 /usr/local/buildpack/tools/v2/python.sh in 16 seconds

root@0b204777c3ec:/# SKIP_VERSION=1 install-tool python 3.11.2
installing v2 tool python v3.11.2
linking tool python v3.11.2
Installed v2 /usr/local/buildpack/tools/v2/python.sh in 11 seconds
```

**Review by viceice — APPROVED** [review]

**Pull request merged**